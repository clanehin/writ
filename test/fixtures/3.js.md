```js
//:: one
```

```js
//== one

// I'm from one
//:: two
```

```js
//== two

// I'm from two
```

