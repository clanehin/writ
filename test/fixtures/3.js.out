// I'm from one
// I'm from two
